docs/installation/autoware/source-installation.md

```diff
@@ -65,9 +65,9 @@ sudo apt-get -y install git
 
 ## How to set up a workspace
 
-!!! info [Using Autoware Build GUI](#using-autoware-build-gui)
+!!! info "[Using Autoware Build GUI](#using-autoware-build-gui)"
 
-    If you prefer a graphical user interface (GUI) over the command line for launching and managing your simulations, refer to the `Using Autoware Build GUI` section at the end of this document for a step-by-step guide.
+      If you prefer a graphical user interface (GUI) over the command line for launching and managing your simulations, refer to the `Using Autoware Build GUI` section at the end of this document for a step-by-step guide.
```

Comment:
```suggestion
     If you prefer a graphical user interface (GUI) over the command line for launching and managing your simulations, refer to the `Using Autoware Build GUI` section at the end of this document for a step-by-step guide.
```

Where do these whitespaces come from? :confused:

Reply:
I'm not entirely sure, but i'll just take the commit in and hope it stays that way